purchases/src/defaults/kotlin/com/revenuecat/purchases/listenerConversions.kt

```diff
@@ -205,6 +219,27 @@ fun Purchases.syncAttributesAndOfferingsIfNeededWith(
     syncAttributesAndOfferingsIfNeeded(syncAttributesAndOfferingsListener(onSuccess, onError))
 }
 
+/**
+ * Note: This method only works for the Amazon Appstore. There is no Google equivalent at this time.
+ * Calling from a Google-configured app will always return AmazonLWAConsentStatus.UNAVAILABLE.
+ *
+ * Get the Login with Amazon consent status for the current user. Used to implement one-click
+ * account creation using Quick Subscribe.
+ *
+ * For more information, check the documentation:
+ * https://developer.amazon.com/docs/in-app-purchasing/iap-quicksubscribe.html
+ *
+ * @param [onSuccess] Called when the consent status was successfully fetched.
+ * @param [onError] Called when there was an error fetching the consent status.
+ */
+@Suppress("unused")
+fun Purchases.getAmazonLWAConsentStatus(
```

Comment:
Usually for these, we rename the method with the `With` suffix like:
```suggestion
fun Purchases.getAmazonLWAConsentStatusWith(
```